src/libs/tmdk/jest.config.js

```diff
@@ -6,4 +6,12 @@ module.exports = {
   testEnvironment: "node",
   testRegex: "/tests/.*\\.(test|spec)?\\.(ts|tsx)$",
   moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+  coverageThreshold: {
+    global: {
+      statements: 80,
+      branches: 80,
+      functions: 80,
+      lines: 80,
```

Comment:
Is this too high for an alpha product? (maybe start with 50/60% if at all?)

Is there a way for us to emphasize integration test coverage instead of unit test coverage?

Reply:
we want to have coverage threshold for every package - at a minimum, i want my team to be able to upgrade dependencies for security and package health reasons with 100% confidence nothing breaks. My question for you is, what code coverage is necessary to enable an away team to perform routine package updates without concern that something may regress?

Reply:
This is from appkit's coverage targets. The tests do not fail if we do not meet these targets, it just gives a warning. We can discuss offline about realistic coverage targets.

Reply:
Thanks Brian, given that this tool is in an "early adoption / alpha phase" I'd like our acceptance testing to essentially be driven by correctness as defined through our end-to-end functional tests. Let's discuss offline on the coverage targets